tasks/publish-to-mrrc/publish-to-mrrc.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: publish-to-mrrc
+spec:
+  params:
+    - name: mrrcParamFilePath
+      description: path of the env file for mrrc parameters to use
+      type: string
+    - name: cosignPubKeyConfig
+      description: the configmap name for cosign verify public key
+      type: string
+    - name: charonAWSSecret
+      description: the secret name for charon aws credential file
+      type: string
+  steps:
+    - name: verify-repo-signature
+      image: registry.redhat.io/rhtas/cosign-rhel9:1.0.2
```

Comment:
please use a sha tag. We don't like floating tags in tasks because we have no guarantee we get the same one each time and a bad push to the floating tag could break users

Reply:
Sure will fix.

Reply:
Fixed. Use a sha256 tag now.